app/lib/frontend/templates/views/pkg/screenshots.dart

```diff
@@ -60,11 +63,15 @@ d.Node collectionsIcon() {
 }
 
 d.Node screenshotThumbnailNode(
-    String thumbnailUrl, List<String>? screenshotUrls) {
+  String thumbnailUrl,
+  List<String>? screenshotUrls,
+  List<String>? screenshotDescriptions,
```

Comment:
Nit: I think at this point all of these could be named parameters.

Reply:
done